config/repo-config.csv

```diff
@@ -1,4 +1,4 @@
-Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List,Ignore Authors List,Shallow Cloning
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commit List,Ignore Authors List,Shallow Cloning
```

Comment:
Seems that we should use `Ignore Commits List, Ignore Authors List` or `Ignore Commit List, Ignore Author List` for consistency.

Reply:
Hi @dcshzj, may I know your opinion on this, thanks!

Reply:
@FH-30 Let's standardise to use the plural instead, as suggested by @fzdy1914.

Reply:
Alright, can!

Reply:
How about standalone configs? I also noticed that they use ```ignore commit list``` instead of ```ignore commits list```. Is it necessary to change those too? @dcshzj @fzdy1914

Reply:
@FH-30 Yep, it is a breaking change for external users, so we will need to include a special note when releasing 1.14. Let's go ahead with standardizing to use "Ignore Commit List" as singular.

Reply:
@dcshzj @fzdy1914 I have made the changes to the standalone config but all the system tests are now failing, I suspect it is because the standalone configs in testrepo alpha till delta uses ```ignoreCommitList``` and ```ignoreAuthorList``` but the change in this PR expects it to be ```ignoreCommitsList``` and ```ignoreAuthorsList``` instead leading to ignorecommitlist and ignoreauthorlist for these repos being null when processed by RepoSense

Reply:
I have managed to get it passing locally with updated standalone configs for testrepo-alpha and testrepo-delta. I'll send PRs in those two repositories and I'll attach them here for reference.

Reply:
The relevant PRs that need to be merged to get tests in this PR passing:
[PR for testrepo-Alpha](https://github.com/reposense/testrepo-Alpha/pull/11)
[PR for testrepo-Delta](https://github.com/reposense/testrepo-Delta/pull/6)